2020-11-10 19:11:20 [[31merror[39m]: code: 404;
 message: Post not found;
 error-stack: Error: Post not found
    at Object.getAllEntities (/home/max/Work/rs/myApp/server/src/utils/db.ts:17:9)
    at Object.getAll (/home/max/Work/rs/myApp/server/src/resources/users/user.repository.ts:9:21)
    at UserService.getAll (/home/max/Work/rs/myApp/server/src/resources/users/user.service.ts:6:32)
    at _router.get (/home/max/Work/rs/myApp/server/src/resources/users/user.router.ts:24:52)
    at Layer.handle [as handle_request] (/home/max/Work/rs/myApp/node_modules/express/lib/router/layer.js:95:5)
    at next (/home/max/Work/rs/myApp/node_modules/express/lib/router/route.js:137:13)
    at Route.dispatch (/home/max/Work/rs/myApp/node_modules/express/lib/router/route.js:112:3)
    at Layer.handle [as handle_request] (/home/max/Work/rs/myApp/node_modules/express/lib/router/layer.js:95:5)
    at /home/max/Work/rs/myApp/node_modules/express/lib/router/index.js:281:22
    at Function.process_params (/home/max/Work/rs/myApp/node_modules/express/lib/router/index.js:335:12);

2020-11-10 19:11:47 [[31merror[39m]: code: 500;
 message: Internal Server Error;
 error-stack: undefined;

2020-11-10 19:40:45 [[31merror[39m]: [Unhandled Rejection] Error: Oops!
    at App.initializeRouters (/home/max/Work/rs/myApp/server/src/app.ts:29:20)
    at new App (/home/max/Work/rs/myApp/server/src/app.ts:21:10)
    at Object.<anonymous> (/home/max/Work/rs/myApp/server/src/server.ts:5:13)
    at Module._compile (internal/modules/cjs/loader.js:778:30)
    at Module.m._compile (/home/max/Work/rs/myApp/node_modules/ts-node/src/index.ts:1043:23)
    at Module._extensions..js (internal/modules/cjs/loader.js:789:10)
    at Object.require.extensions.(anonymous function) [as .ts] (/home/max/Work/rs/myApp/node_modules/ts-node/src/index.ts:1046:12)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3)
